packages/darklang/prettyPrinter/common.dark

```diff
@@ -16,4 +16,21 @@ module Darklang =
     let sign (s: PACKAGE.Darklang.Stdlib.Sign) : String =
       match s with
       | Positive -> ""
-      | Negative -> "-"
\ No newline at end of file
+      | Negative -> "-"
+
+    let escapeSpecialCharacters (s: String) : String =
```

Comment:
Is this file the right place for these functions?

Reply:
Sure.